### ISSUE TITLE
Is correcting the way form ID's named worth spending time on?

### ISSUE THREAD

**Issue by alanmels** [backdrop/backdrop-issues#4706]
I find it inconsistent the way some form ids are named. For example, we have `post_node_form` for Post content type nodes and `comment_node_post_form` for its comments. Should've they been named either:

```
post_node_form
comment_post_node_form
```

or

```
node_post_form
comment_node_post_form
```

Haven't checked how Drupal forms' ids named, but most probably we inherited this mess from Drupal. It's really minor nuance that could be ignored, but then maybe consensus says we should correct it?

**Comment by stpaultim** [comment]
That makes sense to me. Sounds like something that might have to wait for 2.0 or do you think it would be possible to fix this without breaking backwards compatibility?

**Comment by alanmels** [comment]
I believe if it's fixed everywhere in the core, then no functionality should break. It's not likely that some Backdrop projects relied on those particular id namings. Most of the custom alterations operate variables like `$form_id` anyway, so functionality should be intact.

**Comment by indigoxela** [comment]
This is definitely a 2.x task.

Think of all the potentially broken hook_form_FORM_ID_alter() in contrib and custom modules.

**Comment by BWPanda** [comment]
I agree with the suggestion that naming should be consistent, but it will cause breaking changes as @indigoxela suggested.
